src/actions/lat_lon_accel.jl

```diff
@@ -10,6 +10,8 @@ struct LatLonAccel
     a_lat::Float64
     a_lon::Float64
 end
+LatLonAccel(alat::Float64, alon::LaneFollowingAccel) = LatLonAccel(alat, alon.a)
```

Comment:
hacky, but it was already like this so I guess it is ok if we don't want to break things now

Reply:
fair point, but yes I sticked with was there. Is it worth introducing a LatLonAction as discussed? We could forward `propagate(action::LatLonAction{A,B}) = propagate(LatLonAccel(action.lat, action.lon))` or similar